### PR TITLE
Update System.Collections.Immutable version

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: test/EntityFramework.Commands.FunctionalTests/project.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "EntityFramework.Commands": "7.0.0-*",
     "EntityFramework.Core.FunctionalTests": "7.0.0-*",
     "EntityFramework.MicrosoftSqlServer": "7.0.0-*",
+    "System.Collections.Immutable": "1.1.37",
```
Comment (divega) [inline test/EntityFramework.Commands.FunctionalTests/project.json]: I don't understand why this is now needed or what it has to be this version (the EF runtime depends on 1.1.36).

Comment (divega) [inline test/EntityFramework.Commands.FunctionalTests/project.json]: Anyway, thanks for fixing if this was blocking the build.

Comment (pranavkm) [inline test/EntityFramework.Commands.FunctionalTests/project.json]: DNX picked up a new set of Roslyn packages earlier today. Perhaps that caused this issue?

Comment (divega) [inline test/EntityFramework.Commands.FunctionalTests/project.json]: Makes sense. But it is weird. The reference wasn't even here before (was it picked up transitively from the runtime?) And the runtime still depends on 1.1.36, I think on purpose.

@bricelam should probably take a look when possible.

Again, thanks!

**Pull request merged**